modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -268,7 +378,31 @@ class="jetpack-search-filters-widget__search-box-enabled"
 						name="<?php echo esc_attr( $this->get_field_name( 'search_box_enabled' ) ); ?>"
 						<?php checked( $search_box_enabled ); ?>
 					/>
-					<?php esc_html_e( 'Show search box' ); ?>
+					<?php esc_html_e( 'Show search box', 'jetpack' ); ?>
+				</label>
+			</p>
+			<p>
+				<label>
+					<input
+						type="checkbox"
+						class="jetpack-search-filters-widget__sort-controls-enabled"
+						name="<?php echo esc_attr( $this->get_field_name( 'user_sort_enabled' ) ); ?>"
+						<?php checked( $user_sort_enabled ); ?>
+					/>
+					<?php esc_html_e( 'Show sorting controls', 'jetpack' ); ?>
+				</label>
+			</p>
+
+			<p>
+				<label>
+					<?php esc_html_e( 'Default sort order:', 'jetpack' ); ?>
+					<select name="<?php echo esc_attr( $this->get_field_name( 'sort' ) ); ?>" class="widefat">
+		 				<?php foreach( $this->get_sort_types() as $sort_type => $label ) { ?>
+							<option value="<?php echo $sort_type; ?>" <?php selected( $sort, $sort_type ); ?>>
```

Comment:
I know we're using keys we defined our selves, but we should probably `esc_attr()` here, which will make it easier for others to ensure this is escaped when they look at it in the future.